src/Sylius/Bundle/ApiBundle/StateProvider/Shop/OrderItemAdjustmentsProvider.php

```diff
@@ -25,7 +25,7 @@ public function __construct(
     ) {
     }
 
-    public function provide(Operation $operation, array $uriVariables = [], array $context = []): Collection|array
```

Comment:
we should use ecs on related to the PR changes to prevent it from growing too big, put it in separate PR next time 🤔